### ISSUE TITLE
SRW - range in the parameters of the reports causes all reports to recalculate (Source page)

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#602]
Change of the range in the reports' menus on the Source page should not cause all reports to recalculate.
Related to #175.

**Comment by mrakitin** [comment]
All `simulation.` should be removed here:
```json
        "sourceIntensityReport": {
            "title": "Intensity Report",
            "basic": [],
            "advanced": [
                ["Main", [
                    "distanceFromSource",
                    "photonEnergy",
                    "polarization",
                    "characteristic",
                    "fieldUnits",
                    "simulation.samplingMethod",
                    "simulation.sampleFactor",
                    [
                        ["Horizontal", [
                            "simulation.horizontalPosition",
                            "simulation.horizontalRange",
                            "simulation.horizontalPointCount"
                        ]],
                        ["Vertical", [
                            "simulation.verticalPosition",
                            "simulation.verticalRange",
                            "simulation.verticalPointCount"
                        ]]
                    ]
                ]],
                ["Accuracy", [
                    "magneticField",
                    "precision"
                ]],
                ["Scale", [
                    "intensityPlotsWidth",
                    "intensityPlotsScale"
                ]]
            ]
        },
```
Should be implemented like here - https://github.com/radiasoft/sirepo/commit/f7442e9bb7b59e238a971210dc07ed6ca98c46c4.

**Comment by mrakitin** [comment]
Should be fixed before the demo. Working on it now.

**Comment by mrakitin** [comment]
Similar fixes - https://github.com/radiasoft/sirepo/commit/78491fc51772469c517528e7a1609691438298fc and https://github.com/radiasoft/sirepo/commit/f7442e9bb7b59e238a971210dc07ed6ca98c46c4.

**Comment by mrakitin** [comment]
Fixed in 91d2abe.
